tests/utils.go

```diff
@@ -3527,7 +3527,8 @@ func UpdateKubeVirtConfigValue(kvConfig v1.KubeVirtConfiguration) *v1.KubeVirt {
 	}
 
 	suiteConfig, _ := GinkgoConfiguration()
-	if suiteConfig.ParallelTotal > 1 {
+	// ginkgo runs all the specs with the Serial decorator in process #1.
+	if (suiteConfig.ParallelTotal > 1) && (GinkgoParallelProcess() != 1) {
```

Comment:
I am not clear why there is a need to check `suiteConfig.ParallelTotal > 1` now.
If it is equal to 1, won't `GinkgoParallelProcess()` return 1 as well?

Reply:
I can't tell for sure. The ginkgo documentation does not deal with what is the process ID when running serially. I'm not sure we can count on it.

Reply:
Fair enough. Thanks.

Reply:
Fixed. Thanks!